src/FilteredSearch.tsx

```diff
@@ -1,9 +1,11 @@
-import styled from 'styled-components'
 import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import {COMMON, get, SystemCommonProps} from './constants'
 import theme from './theme'
-import {COMMON, get} from './constants'
+import {ComponentProps} from './utils/types'
+import sx, {SxProp} from './sx'
 
-const FilteredSearch = styled.div`
+const FilteredSearch = styled.div<SystemCommonProps & SxProp>`
```

Comment:
Can we make a `FilteredSearchProps` type, for consistency with what we’ve done elsewhere?

Reply:
We've only been pulling out those types when they get long (mentioned in the [google doc](https://docs.google.com/document/d/13U7oryeaPSLWlJxw6y1HPaSnrJ_SwhFiR9SG9JlYYfQ/edit#heading=h.n7hul7xn5z3x)). I think I'll have to rethink how we deal with styled-components props because we're running into problems with `as` prop. So I'll probably update this pattern in all the components soon.